presto-docs/src/main/sphinx/prestissimo/prestissimo-limitations.rst

```diff
@@ -0,0 +1,19 @@
+=======================
+Prestissimo Limitations
+=======================
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Functions
+=========
+
+reduce_agg
+----------
+
+In Prestissimo, ``reduce_agg`` is not permitted to return ``null`` in either the 
+``inputFunction`` or the ``combineFunction``. In Presto Java, this is permitted 
+but undefined behavior. For more information about ``reduce_agg`` in Presto, 
+see `reduce_agg <https://prestodb.io/docs/current/functions/aggregate.html#reduce_agg>`_. 
```

Comment:
Can this use a relative URL?

Reply:
Good catch! Yes it can. 

Edited RST file locally to use relative URL (../functions/aggregate.html#reduce_agg), made a new local build, and tested link with relative URL. 
* The link displays correctly on the page.
* Selecting the link takes me to the correct destination. 

Updated PR and squashed commits.